
# HG changeset patch
# User Phil Thompson <phil@riverbankcomputing.com>
# Date 1701690871 0
# Node ID afc99fa84d0d23f3aad2de78db41b93527568ab5
# Parent  67e0294b505cfb04f94ac67897ed1c7366aab259
Removed a warning message (soon to be an error message) in the v12 module.

diff -r 67e0294b505c -r afc99fa84d0d sipbuild/module/source/12/siplib.c
--- a/siplib.c	Thu Nov 30 21:16:18 2023 +0000
+++ b/siplib.c	Mon Dec 04 11:54:31 2023 +0000
@@ -3896,7 +3896,7 @@
     {
     case '#':
             /* A ctor has an argument with the /Transfer/ annotation. */
-            *selfp = va_arg(va, PyObject *);
+            *selfp = (sipSimpleWrapper *)va_arg(va, PyObject *);
             break;
 
     case 'B':

